Library/Homebrew/os.rb

```diff
@@ -13,6 +13,14 @@ def self.linux?
     RbConfig::CONFIG["host_os"].include? "linux"
   end
 
+  def self.kernel_version
+    @kernel_version ||= if output = Utils.popen_read("uname", "-r").chomp.presence
+      Version.new output
+    else
+      Version::NULL
```

Comment:
Doesn't `Version.create` return `Version::NULL`?

Reply:
That would be `Version.parse` but that might be excessive in this case since `uname -r` already returns a valid version string